templates/Admin/Product/Show/_pricing.html.twig

```diff
@@ -0,0 +1,50 @@
+{% import "@SyliusAdmin/Common/Macro/money.html.twig" as money %}
+{{  dump(sylius_meta.version)}}
+<div id="pricing">
+    <h4 class="ui top attached large header">{{ 'sylius.ui.pricing'|trans }}</h4>
+    <div class="ui attached segment">
+        <table id="pricing" class="ui very basic celled table">
+            <thead>
+                <tr>
+                    <th><strong class="gray text">{{ 'sylius.ui.channels'|trans }}</strong></th>
+                    <th><strong class="gray text">{{ 'sylius.ui.price'|trans }}</strong></th>
+                    <th><strong class="gray text">{{ 'sylius.ui.original_price'|trans }}</strong></th>
+                    {% if sylius_meta.version >= "1.12" %}
```

Comment:
`sylius_meta.version` can be anything eg. `1.12.4-DEV`. It works because PHP cast string to an int however I would compare versions more explicitly with a `split` filter for example :thinking:

Reply:
I'd be for creating a custom twig function with proper semver comparison rather than relying on some weird string manipulation. 🤷